test/system/850-compose.bats

```diff
@@ -55,8 +55,16 @@ EOF
     CONTAINERS_CONF_OVERRIDE=$compose_conf run_podman 42 compose fail
 
     # Make sure the three env variables are set (and parsed)
+    url=".*/podman.sock"
+    # if we run remote with --url check the url arg is honored
+    if [[ "$PODMAN" =~ "--url" ]]; then
+        # get the url from the podman string
+        url="${PODMAN##*--url }"
+        url="${url%% *}"
```

Comment:
This sort of thing, in bash, can be fragile. I can envision ending up with an empty `$url` variable somehow. (Unlikely in this case, but maybe not if someone tweaks this code in a few months). Friendly suggestion for tightening:
```diff
diff --git a/test/system/850-compose.bats b/test/system/850-compose.bats
index 0aacb5c4c..3e6f082b5 100644
--- a/test/system/850-compose.bats
+++ b/test/system/850-compose.bats
@@ -55,18 +55,20 @@ EOF
     CONTAINERS_CONF_OVERRIDE=$compose_conf run_podman 42 compose fail
 
     # Make sure the three env variables are set (and parsed)
+    op='=~'
     url=".*/podman.sock"
     # if we run remote with --url check the url arg is honored
     if [[ "$PODMAN" =~ "--url" ]]; then
         # get the url from the podman string
         url="${PODMAN##*--url }"
         url="${url%% *}"
+        op='='
     fi
-    # podman-remote test might run with --url so unset this because the socket wil be used otherwise
+    # podman-remote test might run with --url so unset this because the socket will be used otherwise
     CONTAINERS_CONF_OVERRIDE=$compose_conf run_podman compose env
-    is "${lines[0]}" "$url"
-    is "${lines[1]}" "0"
-    is "${lines[2]}" ""
+    assert "${lines[0]}" $op "$url" "line 1 of 3"
+    assert "${lines[1]}" = "0"      "line 2 of 3"
+    assert "${lines[2]}" = ""       "line 3 of 3"
 
     DOCKER_HOST="$random_data" DOCKER_CONFIG="$random_data" CONTAINERS_CONF_OVERRIDE=$compose_conf run_podman compose env
     is "${lines[0]}" "$random_data"
```

Reply:
applied